src/items.rs

```diff
@@ -2357,20 +2371,38 @@ fn rewrite_args(
                 ArgumentKind::Variadic(..) => Some("...".to_owned()),
             },
             comment_span_start,
-            span.hi(),
+            next_span_start,
             false,
         );
 
         arg_items.extend(more_items);
     }
 
+    let arg_items_len = arg_items.len();
     let fits_in_one_line = !generics_str_contains_newline
         && (arg_items.is_empty()
-            || arg_items.len() == 1 && arg_item_strs[0].len() <= one_line_budget);
+            || arg_items_len == 1 && arg_item_strs[0].len() <= one_line_budget);
+
+    let has_second_pre_comment = arg_items_len > 1 && arg_items[1].pre_comment.is_some();
```

Comment:
Can you describe what this does and why it's needed ?

Reply:
Please see this [link](https://travis-ci.com/rust-lang/rustfmt/jobs/180096057)

Second arg pre comment also includes first arg post comment in linux environment.
So needs to remove second pre comment if first arg post comment exists.

Reply:
I added the comment to source code.